### DISCUSSION TITLE
Add rabbitmq exporter

### DISCUSSION THREAD

**Discussion by apolzek** [prometheus-operator/kube-prometheus#692] [General]
Assuming I already have the monitoring stack installed and running, how do I add an exporter like the rabbit through Helm ? Do I need to change the stack or just set a parameter in the rabbit exporter?

**Comment by ccrvlh** [comment]
@apolzek  did you manage to find a solution to this?

**Reply by metalmatze** [reply]
I'd say this problem/question is orthogonal to kube-prometheus itself.
This project isn't about helm and even then exporters can always be installed outside of kube-prometheus. 
You'd need a helm chart that deploys the correct objects like ServiceMonitors etc.
